react/helpers/tracker.jsx

```diff
@@ -87,8 +87,8 @@ export const configureTracker = (options = {}) => {
   const root = document.querySelector('[role=application]')
 
   if (root && root.dataset) {
-    appName = readCozyDataFromDOM('appName')
-    cozyDomain = readCozyDataFromDOM('cozyDomain')
+    appName = readCozyDataFromDOM('app').name
+    cozyDomain = readCozyDataFromDOM('domain')
```

Comment:
We should do something more backward compatible : 

readCozyDataFromDOM('appName') || readCozyDataFromDOM('app').name etc.